### ISSUE TITLE
bug report: AppStudioPlugin.TeamsAppCreateFailed

### ISSUE THREAD

**Issue by jke-r** [OfficeDev/microsoft-365-agents-toolkit#8577]
I have a very urgent issue. We have developed an app for Microsoft Teams. This app is already live and used by multiple users in different tenants. We wanted to upload a new version of our app for our clients, but since the latest upgrade of Teams Toolkit FX we are not able to do so anymore. Provisioning is not working anymore and we are unable to update our app. This is a very serious matter to us. 

Here is the message from Teams Toolkit FX:

Failed to create Teams app in Developer Portal, due to A Teams app with that id already exists in your organization's app store. Please manually update the app id and try again.

stack:
TeamsAppCreateFailed: Failed to create Teams app in Developer Portal, due to A Teams app with that id already exists in your organization's app store. Please manually update the app id and try again.
	at AppStudioResultFactory.SystemError (<REDACTED: user-file-path>:2:7469791)
	at Object.updateTeamsApp (<REDACTED: user-file-path>:2:7409150)
	at process.processTicksAndRejections (node:<REDACTED: user-file-path>:96:5)
	at async AppManifest.configure (<REDACTED: user-file-path>:2:7388067)
	at async AppManifest.<anonymous> (<REDACTED: user-file-path>:2:6755649)
	at async TeamsfxCore.provision (<REDACTED: user-file-path>:2:6241474)
	at async TeamsfxCore.<anonymous> (<REDACTED: user-file-path>:2:6755649)
	at async FxCore.provisionResourcesOld (<REDACTED: user-file-path>:2:7763975)
	at async FxCore.<anonymous> (<REDACTED: user-file-path>:2:7912245)
	at async exports.ProjectSettingsWriterMW (<REDACTED: user-file-path>:2:7988501)
	at async exports.ContextInjectorMW (<REDACTED: user-file-path>:2:7900697)
	at async FxCore.<anonymous> (<REDACTED: user-file-path>:2:7907698)
	at async exports.VideoFilterAppBlockerMW (<REDACTED: user-file-path>:2:8068227)
	at async exports.ProjectSettingsLoaderMW (<REDACTED: user-file-path>:2:7986948)
	at async exports.ProjectVersionCheckerMW (<REDACTED: user-file-path>:2:7992488)
	at async exports.AadManifestMigrationMW (<REDACTED: user-file-path>:2:7880258)
	at async exports.ProjectConsolidateMW (<REDACTED: user-file-path>:2:7900409)
	at async exports.ProjectMigratorMW (<REDACTED: user-file-path>:2:7949042)
	at async exports.ConcurrentLockerMW (<REDACTED: user-file-path>:2:7882921)
	at async exports.ErrorHandlerMW (<REDACTED: user-file-path>:2:7913698)

Here is our Teams Dev Portal. It is very strange that the portal is empty:

![image](https://user-images.githubusercontent.com/124296670/234968842-abc2b8a7-8a94-4e7b-a144-3c63713d5c1f.png)

When we try to take ownership of the app, dev portal claims the app does not exist:

![image](https://user-images.githubusercontent.com/124296670/234969170-fd1d9ed8-c8a7-4dbf-a8c0-fa8a6810e842.png)

**Comment by tecton** [comment]
Hi @jke-r, are you using v4.2.4 now? Have you tried to install a previous version as a workaround? I'll try to reproduce it on latest version. It seems like a similar issue related to Developer Portal.

**Comment by jke-r** [comment]
Hi @tecton! Yes, we are using v4.2.4. As a workaround, we edited the file projectSettings.json: We just put attribute "provisioned" back to true. Now, we can at least deploy our code to the cloud. Nevertheless, it is a strange bug. I remember that in older version we were able to re-provision over app again and again without a problem.

**Comment by tecton** [comment]
@jke-r Yes, it's strange that re-provision doesn't work and Developer Portal app list is empty, I can successfully re-provision using 4.2.4 by creating a new tab app. We'll further investigate this issue.
